### PR TITLE
Converts the client library dialog into a wizard

### PR DESCRIPTION
**Pull request by etanshaul**
Some plumbing to turn the library flow into a wizard:

Step 1:
image

Step 2 (placeholder for upcoming API management panel):
image

### CODE REVIEW

**Comment by etanshaul** [comment]
looking into if/where unit tests make sense

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java [hunk 1]
```diff
@@ -48,23 +53,34 @@ public AddCloudLibrariesAction() {
...
 
     @Nullable
     @Override
-    protected JComponent createCenterPanel() {
+    protected String getHelpID() {
+      return null;
+    }
+  }
+
+  /** The wizard step encapsulating the client library selection panel */
```
Comment (nkibler) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesWizardAction.java]: nit: period

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java [hunk 2]
```diff
@@ -58,6 +59,7 @@
 
   GoogleCloudApiSelectorPanel(List<CloudLibrary> libraries) {
     this.libraries = libraries;
+    panel.setPreferredSize(new Dimension(800, 600));
```
Comment (nkibler) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java]: I just realized: can this be part of the .form?

Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java]: I wish. But for some unknown reason it renders differently when added to the form! (its wider, and the text doesn't wrap when resizing). No clue why yet.

**Pull request merged**